public/docs/ts/latest/glossary.jade

```diff
@@ -788,11 +767,11 @@ a#jit
     dynamically as the user navigates through the application, typically
     under the control of a [router](#router).
 
-.l-main-section
-<a id="W"></a>
-<a id="X"></a>
-<a id="Y"></a>
-<a id="Z"></a>
+a#W
+a#X
+a#Y
+.l-main-section#Z
+
 :marked
   ## Zone
```

Comment:
Maybe remove this from the Dart version? I don't think we talk about zones, and Dart has its own concept of zones.